src/kali/install/tools.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+### every exit != 0 fails the script
+set -e
+
+echo "Install some common tools for further installation"
+apt-get update
+apt-get install -y host traceroute kali-linux-top10 xauth vim wget net-tools locales bzip2 libc6-i386 libx11-6 \
+    python-numpy #used for websockify/novnc
+apt-get clean -y
+
+echo "generate locales für en_US.UTF-8"
+locale-gen en_US.UTF-8
```

Comment:
`/etc/locale.gen` does not contain any active locale, so nothing is generated. 
`en_US.UTF-8` has to be uncommented in `/etc/locale.gen`

Log output:
```
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = "en_US:en",
	LC_ALL = "en_US.UTF-8",
	LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
```